File: kt/main/kotlin/moduleconfig/pushtovar/fs/cfgadapt/BarBfCfgAdapter.kt
package tryout.moduleconfig.pushtovar.fs.cfgadapt

import tryout.moduleconfig.pushtovar.config.AppCfgInfo
import tryout.moduleconfig.pushtovar.fs.BarBfCfgInfo
import tryout.moduleconfig.pushtovar.fs.barBfCfgSrc
import tryout.moduleconfig.pushtovar.fwk.CfgSrcAdaptation

fun barBfCfgAdapter(appCfg: AppCfgInfo): BarBfCfgInfo {
	return BarBfCfgInfo(
        z = appCfg.y,
    )
}

val barBfCfgAdaptation = CfgSrcAdaptation(
	barBfCfgSrc,
	::barBfCfgAdapter,
)

File: rs/pullcfgwithoverride/src/fs/bar_bf_cfg_adapter.rs
use crate::fs::BarBfCfgInfo;
use common::config::AppCfgInfo;

pub fn bar_bf_cfg_adapter(app_cfg: &AppCfgInfo) -> BarBfCfgInfo {
    BarBfCfgInfo { z: app_cfg.y }
}
